web-test-runner.utils.js

```diff
@@ -22,57 +22,18 @@ export const chromium = playwrightLauncher({
         browser.newContext({
             ignoreHTTPSErrors: true,
             permissions: ['clipboard-read', 'clipboard-write'],
+            locale: 'en-US',
         }),
 });
 
-/**
- * @todo Remove this configuration and its usage in the Coveralls CI workflow
- * once the Playwright version mismatch between @web/test-runner-playwright
- * and the installed Playwright version is resolved.
- */
-export const coverallsChromium = playwrightLauncher({
-    product: 'chromium',
-    createBrowserContext: ({ browser }) =>
-        browser.newContext({
-            ignoreHTTPSErrors: true,
-            permissions: ['clipboard-read', 'clipboard-write'],
-        }),
-    launchOptions: {
-        executablePath:
-            '/home/runner/.cache/ms-playwright/chromium-1148/chrome-linux/chrome',
-        headless: true,
-    },
-});
-
 export const chromiumWithMemoryTooling = playwrightLauncher({
     product: 'chromium',
+    concurrency: 1,
     createBrowserContext: ({ browser }) =>
         browser.newContext({
             ignoreHTTPSErrors: true,
             permissions: ['clipboard-read', 'clipboard-write'],
-        }),
-    launchOptions: {
-        headless: false,
-        args: [
-            '--js-flags=--expose-gc',
-            '--headless=new',
-            /**
-             * Cause `measureUserAgentSpecificMemory()` to GC immediately,
-             * instead of up to 20s later:
-             * https://web.dev/articles/monitor-total-page-memory-usage#local_testing
-             **/
-            '--enable-blink-features=ForceEagerMeasureMemory',
-        ],
-    },
-});
-
-export const chromiumWithMemoryToolingCI = playwrightLauncher({
```

Comment:
no need for two different launchers. i set up a memory config similar to our other ci configs that lowers the concurrency for CI